http-server-netty/src/main/java/io/micronaut/http/server/netty/AbstractBufferingHttpContentProcessor.java

```diff
@@ -1,113 +0,0 @@
-/*
- * Copyright 2017-2020 original authors
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * https://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package io.micronaut.http.server.netty;
-
-import io.micronaut.core.annotation.Internal;
-import io.micronaut.core.async.processor.SingleThreadedBufferingProcessor;
-import io.micronaut.http.exceptions.ContentLengthExceededException;
-import io.micronaut.http.netty.stream.StreamedHttpMessage;
-import io.micronaut.http.server.HttpServerConfiguration;
-import io.netty.buffer.ByteBufHolder;
-import io.netty.handler.codec.http.multipart.HttpData;
-import org.reactivestreams.Subscriber;
-
-import java.util.concurrent.atomic.AtomicLong;
-
-/**
- * Abtract implementation of the {@link HttpContentProcessor} interface that deals with limiting file upload sizes.
- *
- * @param <T> The type
- * @author Graeme Rocher
- * @since 1.0
- */
-@Internal
-public abstract class AbstractBufferingHttpContentProcessor<T> extends SingleThreadedBufferingProcessor<ByteBufHolder, T> implements HttpContentProcessor<T> {
```

Comment:
This is used in Jackson XML, we will need to address that https://github.com/micronaut-projects/micronaut-jackson-xml/blob/1e6f2726de8bcf88cbb200ca7535e3cfa494b901/jackson-xml/src/main/java/io/micronaut/xml/jackson/server/XmlContentProcessor.java

Reply:
https://github.com/micronaut-projects/micronaut-jackson-xml/pull/235